### PR TITLE
PICARD-1220: Add keyboard shortcut to delete scripts.

### PR DESCRIPTION
**Pull request by vishichoudhary**
# Summary

* This is a…
    * [ ] Bug fix
    * [x] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PICARD-1220](https://tickets.metabrainz.org/browse/PICARD-1220)

# Solution
In options > Scripting. On pressing delete key, the script will be removed.

# Action

### CODE REVIEW

**Comment by vishichoudhary** [comment]
previous discussion can be found on https://github.com/metabrainz/picard/pull/874

File: picard/ui/options/scripting.py [hunk 1]
```diff
@@ -232,6 +232,14 @@ def __init__(self, parent=None):
         self.last_selected_script_pos = 0
         self.ui.splitter.setStretchFactor(0, 1)
         self.ui.splitter.setStretchFactor(1, 2)
+        self.shortcut = QtWidgets.QShortcut(self.ui.script_list)
+        self.shortcut.setKey(QtGui.QKeySequence.Delete)
+        self.shortcut.activated.connect(self.script_remove_processor)
```
Comment (sambhav) [inline picard/ui/options/scripting.py]: Rename this to `delete_selected_script`

Comment (sambhav) [reply picard/ui/options/scripting.py]: @vishichoudhary  I meant the connected function. The shortcut variable itself could be `self.delete_shortcut`.

File: picard/ui/options/scripting.py [hunk 2]
```diff
@@ -232,6 +232,14 @@ def __init__(self, parent=None):
         self.last_selected_script_pos = 0
         self.ui.splitter.setStretchFactor(0, 1)
         self.ui.splitter.setStretchFactor(1, 2)
+        self.delete_selected_script = QtWidgets.QShortcut(self.ui.script_list)
+        self.delete_selected_script.setKey(QtGui.QKeySequence.Delete)
+        self.delete_selected_script.activated.connect(self.script_remove_processor)
```
Comment (sambhav) [inline picard/ui/options/scripting.py]: @vishichoudhary  I meant the connected function. The shortcut variable itself could be `self.delete_shortcut`.

Comment (vishichoudhary) [reply picard/ui/options/scripting.py]: Shouldn't say this but. naming the variables to seem most difficult task for me. :-( 😅

**Pull request merged**